src/org/zaproxy/zap/extension/websocket/WebSocketMessageDTO.java

```diff
@@ -194,7 +195,24 @@ public String getReadablePayload() throws InvalidUtf8Exception {
 			return "";
 		}
 	}
-	
+
+	/**
+	 * Returns content of {@link WebSocketMessageDTO#payload} directly if it is
+	 * of type {@link String}. Otherwise tries to convert that by replacing unmapped
+	 * malformed characters.
+	 * @return readable representation of payload
+	 * @throws InvalidUtf8Exception
```

Comment:
Does it no longer throw or is the method declaration/signature wrong?

Reply:
That's right. Thank you @kingthorin